packages/editor/src/components/editor-interface/index.js

```diff
@@ -160,7 +160,7 @@ export default function EditorInterface( {
 
 					<EditorContentSlotFill.Slot>
 						{ ( [ editorCanvasView ] ) =>
-							! isPreviewMode && editorCanvasView ? (
+							editorCanvasView ? (
```

Comment:
The only "views" are revisions and style book.

Style book is the only "view" that can be activated in edit and view mode.

This PR reverts the functionality to how it was before. This condition could be tightened up in this PR to check whether the style book is open too.

@youknowriad gave some great advice over in https://github.com/WordPress/gutenberg/pull/62146/files#r1621762342 about how the whole view system could be refactored. I haven't digested it all yet, but maybe things could be improved iteratively and we just fix the regression here as it is.

Reply:
That change is fine by me 👍